File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int A = scanner.nextInt();
        int B = scanner.nextInt();
        int C = scanner.nextInt();

        String answer = "Error";
        if (A + B + C == 180) {
            if (A == B && B == C) {
                answer = "Equilateral";
            } else {
                answer = (A == B || A == C || B == C) ? "Isosceles" : "Scalene";
            }
        }
        System.out.println(answer);
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
A = int(input())
B = int(input())
C = int(input())

answer = "Error"
if (A + B + C) == 180:
    if A == B == C:
        answer = "Equilateral"
    else:
        answer = "Isosceles" if A == B or A == C or B == C else "Scalene"
print(answer)